app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMessage } from '../store/message/messageSlice';
+
+const Greeting = () => {
+  const message = useSelector((state) => state.message.message);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getMessage());
+  }, []);
+
+  return <h1>{message.message}</h1>;
```

Comment:
- [ ] This is the message I receive when I visit your root path. It should display the fetched message. Kindly fix that.

  ![req](https://user-images.githubusercontent.com/36542986/190162381-d84657f0-7e96-4822-a166-ff44130efdcf.jpg)

  I think it may be related to this:

  ![ee](https://user-images.githubusercontent.com/36542986/190164341-6ffa3a86-82ec-4a87-b1de-f6ecc26cded5.jpg)